### PR TITLE
feat(test): better formatting for test elapsed time

### PR DESCRIPTION
**Pull request by bartlomieju**
Closes https://github.com/denoland/deno/issues/12609

### CODE REVIEW

File: cli/tools/test.rs [hunk 1]
```diff
@@ -260,6 +260,22 @@ impl PrettyTestReporter {
...
+  if elapsed < 1_000 {
+    return format!("{}ms", elapsed);
+  }
+  if elapsed < 1_000 * 60 {
+    return format!("{}s", (elapsed / 1000).floor());
+  }
+
+  let seconds = (elapsed / 1_000).floor();
+  let minutes = (seconds / 60).floor();
+  let seconds_reminder = seconds - (minutes * 60);
```
Comment (dsherret) [inline cli/tools/test.rs:275]: `second % 60`

File: cli/tools/test.rs [hunk 2]
```diff
@@ -249,7 +249,7 @@ impl PrettyTestReporter {
     println!(
       "{} {}",
       status,
-      colors::gray(format!("({}ms)", elapsed)).to_string()
+      colors::gray(human_elapsed(elapsed)).to_string()
```
Comment (dsherret) [inline cli/tools/test.rs:252]: I think this should also be done for test reporting?

**Review by dsherret — APPROVED** [review]
LGTM, but probably needs it for test case reporting too

**Comment by bartlomieju** [comment]
> LGTM, but probably needs it for test case reporting too

Yeah, not a big deal for "ms" and "s", but I don't want to add a test that takes more than a minute just to assert it outputs (1m1s) 😬

**Comment by dsherret** [comment]
@bartlomieju we could unit test `human_elapsed`. I don't think testing the structs at a higher level/integration tests are necessary.

**Comment by bartlomieju** [comment]
> @bartlomieju we could unit test `human_elapsed`. I don't think testing the structs at a higher level/integration tests are necessary.

SGTM, I'll do that.

**Pull request merged**